app/components/autocomplete/autocomplete.tsx

```diff
@@ -106,44 +102,31 @@ const Autocomplete = ({
     const appsTakeOver = showingAppCommand;
     const showCommands = !(showingChannelMention || showingEmoji || showingAtMention);
 
-    const maxListHeight = useMemo(() => {
-        if (maxHeightOverride) {
-            return maxHeightOverride;
-        }
-        const isLandscape = dimensions.width > dimensions.height;
-        let postInputDiff = 0;
-        if (isTablet && postInputTop && isLandscape) {
-            postInputDiff = MAX_LIST_TABLET_DIFF;
-        } else if (postInputTop) {
-            postInputDiff = MAX_LIST_DIFF;
-        }
-        return MAX_LIST_HEIGHT - postInputDiff;
-    }, [maxHeightOverride, postInputTop, isTablet, dimensions.width]);
-
     const wrapperStyles = useMemo(() => {
         const s = [];
         if (Platform.OS === 'ios') {
             s.push(style.shadow);
         }
-        if (isSearch) {
-            s.push(style.base, paddingTop ? {top: paddingTop} : style.searchContainer, {maxHeight: maxListHeight});
-        }
         return s;
-    }, [style, isSearch && maxListHeight, paddingTop]);
+    }, [style]);
 
     const containerStyles = useMemo(() => {
-        const s = [];
-        if (!isSearch && !fixedBottomPosition) {
-            const iOSInsets = Platform.OS === 'ios' && (!isTablet || rootId) ? insets.bottom : 0;
-            s.push(style.base, {bottom: postInputTop + LIST_BOTTOM + iOSInsets});
-        } else if (fixedBottomPosition) {
-            s.push(style.base, {bottom: 0});
+        const s = [style.base];
+        if (growDown) {
+            s.push({top: -position});
+        } else {
+            s.push({bottom: position});
         }
         if (hasElements) {
             s.push(style.borders);
         }
         return s;
-    }, [!isSearch, isTablet, hasElements, postInputTop]);
+    }, [hasElements, position, growDown, style]);
+
+    const isLandscape = dimensions.width > dimensions.height;
+    const postInputDiff = (isTablet && isLandscape) ? MAX_LIST_TABLET_DIFF : MAX_LIST_DIFF;
```

Comment:
perhaps we should rename this variable so that is clear that the autocomplete is not really related to the postInput